### ISSUE TITLE
Accessibility issues: Storefront

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#4018]
This is a summary of findings doing a complete accessibility test on our storefront. 

### Pages tested
- https://www.designsystemet.no/no
- https://www.designsystemet.no/no/fundamentals 
- https://www.designsystemet.no/no/patterns 
- https://www.designsystemet.no/no/blog
- https://theme.designsystemet.no/no

Our components are not a part of this test. 
Tested with Chrome and VoiceOver

#### Focus indicator
Focus indicator on selected `border-radius` is styled differently then the rest of our components. This is breaking Focus Appearance which is not required by law. (WCAG 2.2 AAA) 

<img width="390" height="273" alt="Image" src="https://github.com/user-attachments/assets/d516192f-ebf4-4b9f-b509-f3a77c59af1c" />

- [ ] Focus indicator is now the same on selected and non-selected

#### Use of `<article>`

It seems that we are using `<article>` on our card components. They are not landmarks, but they do show up in the landmark meny in VoiceOver. So for the overview of our documentation is looks like this:

<img width="3444" height="1898" alt="Image" src="https://github.com/user-attachments/assets/69382b50-d287-4f1f-8939-8948260daad4" />

This is not breaking any WCAG guidelines to my knowledge, but I just wanted to check if this is the intended behavior? 

- [ ] Look into if we want to use article on our card components

####  2.5.4 Label in name

We have different visual and accessible names for the “legg til farge” buttons in the theme builder. They start the same, so it’s not a violation, but they don’t use the same wording as we do in the GUI. Maybe change the accessible name to “legg til main farge”? In that case, we’d also need to do the same for support. This is not critical.

<img width="1133" height="521" alt="Image" src="https://github.com/user-attachments/assets/cca90801-e11a-4f8a-b965-fec0803f2c64" />

- [ ] Concider if this is the right thing to do. It might be best to use the norwegian names here. 

#### Navigation cards on the front page read twice
Using VoiceOver/chrome I get the title of the cards read twice. I cant see what causes the issue. Might be a VoiceOver bug? 
<img width="901" height="462" alt="Image" src="https://github.com/user-attachments/assets/11c25869-bfe7-4c17-a155-19db6a38308c" />

- [ ] Is this an issue with our code or a bug?

#### Code example gets keyboard focus
I found this code snippet that gets keyboard focus. The next code snippet do not get focus. This is from [Typography](https://www.designsystemet.no/no/fundamentals/design-tokens/typography)
<img width="800" height="397" alt="Image" src="https://github.com/user-attachments/assets/a7fa1ba2-e2c6-4d09-a388-c611dab91376" />

- [ ] Remove code snippet from tab-order

#### Zoom 200% color table
Works fine on smaller screens, but when zooming we get some issues
<img width="684" height="759" alt="Image" src="https://github.com/user-attachments/assets/566d3be1-48c0-4fa8-bb9a-63710ea7a70b" />

- [ ] Table works with 200% zoom

#### Border-radius type="number"
Border-radius input type is `number` I guess this is correct. But I came across a bug where I could not enter anything when using VoiceOver/Chrome. Maybe not a problem that can be fixed at our end. But might be something to investigate related to  #3373 

#### `Aria-expanded` on Toggle for sidebar in themebuilder
There are no indication for screen readers to say that it is open or glosed right now. 

<img width="888" height="355" alt="Image" src="https://github.com/user-attachments/assets/5495e80e-4dcb-47a3-945a-7a1a5caf3b00" />

- [ ] Added `aria-expanded` or something similar that can indicate what state the toggle is in.

#### Designsystemet link (logo) goes to theme.designsystemet when in themebuilder. 
Seems like we change what the "frontpage" is when going into the themebuilder. When in our documentation our logobutton will send the user to our frontpage (designsystemet.no) but when in the theme builder it will send them to (theme.designsystemet.no) 

- [ ] The top left link (logo) is sending the user to our frontpage no matter what page you are on

**Comment by Barsnes** [comment]
## Navigation cards on the front page read twice
This was not an issue on windows + nvda. Not fixed.

## Use of `<article>`
This was fine. Not fiex.

## Code example gets keyboard focus
This is because it is scrollable. Not fixed.

## Border-radius type="number"
Works on windows + nvda. Not fixed.

## Designsystemet link (logo) goes to theme.designsystemet when in themebuilder.
Not fixed
